### ISSUE TITLE
Publish to Packagist

### ISSUE THREAD

**Issue by Livijn** [helmutschneider/swish-php#15]
We should publish this package to [Packagist](http://packagist.org) so you can easily require this via composer, e.g.: `composer require helmutschneider/swish-php`.

**Comment by helmutschneider** [comment]
Good idea. Do you know what kind of stability requirements Packagist has? I'm not really confident in calling this "stable" 😀

**Comment by Livijn** [comment]
I would suggest just versioning this as 0.1 which implicates a "not-so-stable" release 🤠

**Comment by Livijn** [comment]
Any update? Should just take a couple of minutes ✌️

**Comment by tarreislam** [comment]
hoho! we rather have .1 than dev-master :D :D :D

**Comment by oakleaf** [comment]
Any update on this?

**Comment by tarreislam** [comment]
I have made my own wrapper for swish.

Well, its a wrapper for laravel, I could decouple the package if you guys want.

packagist:

https://packagist.org/packages/tarre/laravel-swish

source code

https://github.com/tarreislam/laravel-swish

**Comment by tarreislam** [comment]
Since they are no docs yet, you can check out the tests to get an idea of how to implement into your laravel apps

https://github.com/tarreislam/laravel-swish/tree/master/tests/Client

**Comment by MPV** [comment]
Maybe you’re already aware, but it seems like @mladjom already has published his fork of this repo to packagist (with currently about 5000 installs so far): https://packagist.org/packages/helmutschneider/swish-php

**Comment by mladjom** [comment]
I pointed it back to main repository.

**Comment by dhedberg** [comment]
@mladjom @helmutschneider It would be nice if we could get the auto-updating to work on packagist.

This might require that @mladjom transfers the maintainer status in packagist to @helmutschneider, if possible.
